minica/minica_test.go

```diff
@@ -23,7 +23,7 @@ func (p badSigner) Public() crypto.PublicKey {
 	return []byte("foo")
 }
 
-func (p badSigner) Sign(r io.Reader, digest []byte, opts crypto.SignerOpts) (signature []byte, err error) {
+func (p badSigner) Sign(_ io.Reader, digest []byte, opts crypto.SignerOpts) (signature []byte, err error) {
```

Comment:
Is the linter not failing with digest and opts?

Reply:
https://github.com/smallstep/workflows/pull/31/files -- stopped using this linter for tests midway through generating the PR.